index-climate.md

```diff
@@ -71,6 +75,10 @@ Tool | Description | Reference
 {% include tool.html id="gdal_ogrinfo" %} |  [Lists information about an OGR-supported data source](https://gdal.org/programs/ogrinfo.html) | -
 {: .table.table-striped .tooltable}
 
+# Machine Learning Workbench
+
+For Machine Learning tools, use the [Galaxy Machine Learning Workbench](https://ml.usegalaxy.eu/). To have access to all your Galaxy histories and data, make sure to login with the same username and password than on the Climate Science Workbench. 
```

Comment:
We can add the machine learning tools to your flavor as well, if you like.
Please also not that technicall the tools are already available, just hidden by default.

Reply:
We will probably add them in a near future (I probably need to make some examples/training with climate data). For now, I would like to make sure our users know many ML tools are available (and excellent training material too!).
Thanks.